features/telemetry/api/src/main/java/org/opennms/netmgt/telemetry/api/receiver/TelemetryMessage.java

```diff
@@ -28,24 +28,37 @@
 
 package org.opennms.netmgt.telemetry.api.receiver;
 
-import org.opennms.core.ipc.sink.api.Message;
 import java.net.InetSocketAddress;
 import java.nio.ByteBuffer;
 import java.util.Date;
 
+import org.opennms.core.ipc.sink.api.Message;
+
 public class TelemetryMessage implements Message {
     private final InetSocketAddress source;
     private final ByteBuffer buffer;
     private final Date receivedAt;
+    private final boolean isFlowMessage;
+    private final boolean useRoutingKey;
 
     public TelemetryMessage(InetSocketAddress source, ByteBuffer buffer) {
-        this(source, buffer, new Date());
+        this(source, buffer, new Date(), false, true);
     }
 
     public TelemetryMessage(InetSocketAddress source, ByteBuffer buffer, Date receivedAt) {
+        this(source, buffer, receivedAt, false, true);
+    }
+
+    public TelemetryMessage(InetSocketAddress source, ByteBuffer buffer, Date receivedAt, boolean isFlowMessage) {
+        this(source, buffer, receivedAt, isFlowMessage, true);
+    }
+
+    public TelemetryMessage(InetSocketAddress source, ByteBuffer buffer, Date receivedAt, boolean isFlowMessage, boolean useRoutingKey) {
         this.source = source;
         this.buffer = buffer;
         this.receivedAt = receivedAt;
+        this.isFlowMessage = isFlowMessage;
```

Comment:
the naming pattern for boolean properties is just `property` for the field and `isProperty` and `setProperty` for getters and setter. So the field should be called `flowMessage` and the getter should be called `isFlowMessage`.

Reply:
Done